src/renderer/src/i18n/en-US/settings.json

```
@@ -262,335 +114,112 @@
-        "noOperationalUrls": "No available operating address found",
-        "pageNumber": "page number",
-        "pageNumberPlaceholder": "Please enter the page number",
-        "serverAlreadyExists": "The server already exists, skip import",
-        "syncComplete": "Synchronous completion",
-        "invalidServerData": "Invalid server data"
-      },
+    "knowledgeBase": {
+      "title": "Knowledge Base Settings",
+      "addKnowledgeBase": "Add Knowledge Base",
+      "selectKnowledgeBaseType": "Please select type of knowledge base to add",
+      "difyDescription": "Dify helps you manage and use document data",
```

Biome (2.1.2)

[error] The key difyDescription was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)